### PR TITLE
Add copyheart license. solves issue #851

### PR DESCRIPTION
**Pull request by starlord1311**
Added the copyheart license.

### CODE REVIEW

File: src/licensedcode/data/licenses/copyheart.LICENSE [hunk 1]
```diff
@@ -0,0 +1,53 @@
+? Copying is an act of love. Please copy and share.
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/copyheart.LICENSE]: This is full of mojibake  https://en.wikipedia.org/wiki/Mojibake ... Can you find a way to get a clean plain text for this? but then.... the text is really only that first line, the rest is not: therefore just get rid of the rest all right IMHO

File: src/licensedcode/data/rules/copyheart_syntax_highlighter.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+licenses:
+    - copyheart
```
Comment (pombredanne) [inline src/licensedcode/data/rules/copyheart_syntax_highlighter.yml]: Add a new line

Comment (pombredanne) [reply src/licensedcode/data/rules/copyheart_syntax_highlighter.yml]: And where is the `copyheart_syntax_highlighter.RULE` rule file?

File: tests/licensedcode/data/licenses/copyheart.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+license:
+    - Copyheart
```
Comment (pombredanne) [inline tests/licensedcode/data/licenses/copyheart.yml]: Add a new lines after and use a lowercase key
Also where is the copyheart.txt test file?

File: src/licensedcode/data/rules/copyheart_syntax_highlighter.yml (hunk 2 above)
Comment (starlord1311) [reply src/licensedcode/data/rules/copyheart_syntax_highlighter.yml]: Didn't get what you meant by add a new line

File: src/licensedcode/data/licenses/copyheart.LICENSE (hunk 1 above)
Comment (starlord1311) [reply src/licensedcode/data/licenses/copyheart.LICENSE]: Didn't get what you meant. Can you please explain

Comment (pombredanne) [reply src/licensedcode/data/licenses/copyheart.LICENSE]: Have you seen all the junk characters in this file?

**Comment by pombredanne** [comment]
Can you make sure that your sign off line is not in the commit subject?
The format is:
```
<subject line> (impertative style, and trailing #issue-number
<blank line>
<extra commit description line(s)>
<blank line>
<signoff line(s)>
```

File: src/licensedcode/data/licenses/copyheart.LICENSE (hunk 1 above)
Comment (starlord1311) [reply src/licensedcode/data/licenses/copyheart.LICENSE]: yeah. @pombredanne so what am i supposed to do with those characters.

Comment (pombredanne) [reply src/licensedcode/data/licenses/copyheart.LICENSE]: You need to find or craft a clean text-only version of this

Comment (starlord1311) [reply src/licensedcode/data/licenses/copyheart.LICENSE]: there are hearts in the text. what am i supposed to do with those?

**Comment by pombredanne** [comment]
I think you fixed the mojibake nicely. Thank you!
Now the main license text should IMHO only be 
`Copying is an act of love. Please copy and share.`
.... e.g. not the full article about the license.

**Comment by starlord1311** [comment]
@pombredanne is it right now?

**Comment by pombredanne** [comment]
`♡ Copying is an act of love. Please copy.`

....  is the ONLY license text you want. NO copyrights. And files should not be binaries but text UTF-8 encoded.


Each line below should be a different rule, UTF-8 no binary
`Copying is an act of love. Please copy.`
`Copying Art is an act of love. Please copy and share.`
`Copying Art is an act of love. Love is not subject to law.`
`Copying Art is an act of love. Please copy and share.`
`Copying is an act of love. Please copy.`
`Copying is an act of love. Love is not subject to law.`

**Comment by pombredanne** [comment]
And BTW your tests still fail

**Comment by pombredanne** [comment]
@starlord1311 Thanks for the updates. But you need to amend your commit messages or squash your commits. "changes" is not a great message and is missing your signoff too.

There are aslo several test failures to address.

**Comment by starlord1311** [comment]
@pombredanne i am not understanding why they are failing. can you please help?

**Comment by pombredanne** [comment]
Did you try to run the tests locally? And check the stack traces?

**Comment by pombredanne** [comment]
Yes, of course

**Comment by pombredanne** [comment]
@starlord1311 Thank you ++ for all your efforts but in the end I went with @yash-nisar #899 that passes the tests alright

**Pull request closed without merging**